src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -186,7 +190,7 @@ public static void main(String[] args) throws Exception {
     public JSONObject getJsonUrl(String url) throws MalformedURLException, IOException {
         try (
                 InputStream is = new URL(url).openStream();
-                InputStreamReader isr = new InputStreamReader(is);
+                InputStreamReader isr = new InputStreamReader(is, "UTF-8");
```

Comment:
This made sense in Java 8, but now that the code base is on Java 11 we can use the more specific type:

```suggestion
InputStreamReader isr = new InputStreamReader(is, StandardCharsets.UTF_8);
```

(Plus `import java.nio.charset.StandardCharsets;` at the top.)

Reply:
Resolved in 48046e678b43302ab5192837ca26cbf36f6ad865